tests/testthat/test-metadata.R

```diff
@@ -167,3 +167,24 @@ test_that("uncertainty metadata set for all model output data", {
   expect_true(all(!(output_meta$error_low_column == "")))
   expect_true(all(!(output_meta$error_high_column == "")))
 })
+
+
+test_that("metadata format column hasn't been messed by Excel", {
```

Comment:
Nice! Good way to catch this. Alternatively we could switch to another data format like yaml to avoid it ever opening in excel